### ISSUE TITLE
Java would like to receive keystrokes prompt on macOS Catalina

### ISSUE THREAD

**Issue by klabarge** [qzind/tray#490]
I received a dialogue prompt on macOS Catalina (beta) `"Java" would like to receive keystrokes from any application` when I used HTML printing.

I was only able to get this dialogue to popup one time. I have not granted Java access, and all printing appears to be fine.

**macOS Version:** Catalina 10.15 Beta (19A536g)
**Java:** AdoptOpenJDK "11.0.4" 2019-07-16

![image](https://user-images.githubusercontent.com/12505463/64829539-b9c6bc80-d59a-11e9-90b6-0ee36125ea75.png)

**Comment by tresf** [comment]
I've updated to macOS Catalina 10.15 Beta (19A558d) but I cannot reproduce.
* I've tested with a notarized release (latest 2.0.11) and cannot reproduce.
* I've tested with a non-notarized release (2.0.11-RC2) and cannot reproduce.
* I've tested with a non-notarized release (2.0.11-RC1) and cannot reproduce.
* I've tested with IntelliJ IDEA and cannot reproduce.

@klabarge is there a chance that macOS Catalina patched something along the way?
